cli/src/commands/account.rs

```diff
@@ -242,6 +254,12 @@ impl Account {
             PrivateKey::try_from(seed).map_err(|_| anyhow!("Failed to convert the seed into a valid private key"))?;
         // Construct the account.
         let account = snarkos_account::Account::<N>::try_from(private_key)?;
+        // Save to file instead of printing it back to the user
+        if let Some(path) = save_to_file {
+            let mut file = File::create(path)?;
+            file.write_all(account.private_key().to_string().as_bytes())?;
+            return Ok(String::from(""));
```

Comment:
Great catch :fishing_pole_and_fish: I've changed the behavior and the test-cases. Now, we will not return early but continue.

Now the code continues like normal. Technically, it `save-to-file` could co-run with discreet. But I don't think it is a realistic use-case. Because one is dependent on the user looking at the screen, the other is for a programmatic solution.